### PR TITLE
Unescape

### PR DESCRIPTION
**Pull request by dlecocq**
Replace all escaped entities with their corresponding byte value.

@b4hand @tammybailey @tanglyh @martin-seomoz

### CODE REVIEW

File: include/url.h [hunk 1]
```diff
@@ -75,24 +75,64 @@ namespace Url
...
             , userinfo_(other.userinfo_) { }
 
-        /**************************
-         * Component-wise access. *
-         **************************/
+        /**************************************
+         * Component-wise access and setting. *
+         **************************************/
         const std::string& scheme() const { return scheme_; }
+        Url& scheme(const std::string& s)
```
Comment (b4hand) [inline include/url.h]: I wouldn't overload accessors with mutators. For mutators, it is common to use a `set` prefix.

Comment (b4hand) [inline include/url.h]: Looks like this comment should have gone to #17.

**Comment by dlecocq** [comment]
Rebased -- any objections?

**Comment by b4hand** [comment]
LGTM

**Comment by tammybailey** [comment]
LOKTM

**Pull request merged**